packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-dart-visitor.test.ts

```diff
@@ -346,6 +346,18 @@ describe('AppSync Dart Visitor', () => {
       const generatedCode = visitor.generate();
       expect(generatedCode).toMatchSnapshot();
     });
+
+    it('Should set fields with @default as nullable', () => {
```

Comment:
Is this the behavior we want vs erroring out? If the schema shows it as non nullable, it could be misleading

Reply:
yes, this is similar to how we handle the ID fields